### PR TITLE
Fixed #26424 -- Added support for schemes='__all__' to URLValidator.

### PR DESCRIPTION
**Pull request by bmispelon**
Thanks to Burhan Khalid for the original pull request.

ticket-26424

### CODE REVIEW

**Comment by felixxm** [comment]
ticket-26424?

**Comment by bmispelon** [comment]
> [ticket-26424](https://code.djangoproject.com/ticket/26424)?

Oops, you're right. That's what I get for not copy/pasting the ticket number :man_facepalming:

File: docs/ref/validators.txt [hunk 1]
```diff
@@ -166,6 +166,12 @@ to, or in lieu of custom ``field.clean()`` methods.
         list is ``['http', 'https', 'ftp', 'ftps']``. As a reference, the IANA
         website provides a full list of `valid URI schemes`_.
 
+        .. versionadded:: 3.1
```
Comment (carltongibson) [inline docs/ref/validators.txt]: Make this a version changed? (Below)

Comment (bmispelon) [reply docs/ref/validators.txt]: Fixed, thanks

**Comment by bmispelon** [comment]
> I kind of have mixed feelings about this. PR looks ≈fine but I'm not quite sure we should be adding this...
> 
> I've subclasses `URLValidator` a number of times over the years to customise `schemes`. It's never really been much of a hardship. I worry that by adding this, we're saying "just use `'__all__'`, when really I'd still want to be saying, "create a custom validator for the known set of schemes that you want to accept".
> 
> Perhaps I'm just getting old...

I don't really have strong feelings about this ticket (I've just been going through old tickets and saw this one with an almost-complete PR).
Maybe @claudep could chime in?

**Comment by felixxm** [comment]
Closing per ticket.

**Pull request closed without merging**